test/framework/src/test/java/org/elasticsearch/test/VersionUtilsTests.java

```diff
@@ -62,22 +62,22 @@ public void testRandomVersionBetween() {
         assertTrue(got.onOrBefore(Version.CURRENT));
 
         // sub range
-        got = VersionUtils.randomVersionBetween(random(), Version.V_6_0_0_alpha1,
-                Version.V_6_2_4);
-        assertTrue(got.onOrAfter(Version.V_6_0_0_alpha1));
-        assertTrue(got.onOrBefore(Version.V_6_2_4));
+        got = VersionUtils.randomVersionBetween(random(), Version.V_7_0_0,
```

Comment:
These tests should use Version.fromId. There shoudln't be any tests in VersionUtilsTests that use version constants because all version constants are essentially ephemeral. Using constants here means they must be updated at some point in the future. The logic of versions can be tested without using constants.